src/components/App/App.js

```diff
@@ -21,13 +53,15 @@ function App() {
 
   return (
     <main className="main-app">
+      {error && <h1 style={{color: 'red'}}>Something went wrong, please try again</h1>}
       {singleMovie ? 
         <>
           <ExitMovie viewAll={viewAll} />
           <SingleMovie selectedMovie={singleMovie} /> 
         </> : 
         <>
           <img className='main-title' src={banner} alt='old school theater banner with the title rancid tomatillos' />
+          {loading && <h2 style={{color: 'red'}}>Loading...</h2>}
```

Comment:
My suggestion is to try to avoid using inline styling as much as possible, as it can make our code appear messy and difficult to read.